internal/command/command.go

```diff
@@ -272,7 +272,7 @@ func checkVersion(logger output.Logger) {
 		return
 	}
 
```

Comment:
```suggestion
	currentVersion := build.Semver()
	if isDevelopment() {
		return // avoid warning in local development
	}

	// If using cadence-v1.0.0 pre-release, check for cadence-v1.0.0 releases instead
	if strings.Contains(currentVersion, "cadence-v1.0.0") {
		checkVersionCadence1(logger)
		return
	}
```

I think this merge conflict didn't get solved correctly, it's missing this part.